### PR TITLE
support generating cobra docs

### PR DESCRIPTION
**Pull request by Okabe-Rintarou-0**
**What type of PR is this?**
enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
implement #1020 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: ctl/common/common.go
Comment (hzxuzhonghu) [inline ctl/common/common.go:1]: m,iss copyright

**Review by hzxuzhonghu — APPROVED** [review]

**Comment by hzxuzhonghu** [comment]
I am thining how can we migrate these docs to [kmesh.net](https://kmesh.net/en/)

File: ctl/common/common.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"github.com/spf13/cobra"
```
Comment (LiZhenCheng9527) [inline ctl/common/common.go:4]: golint error

**Comment by Okabe-Rintarou-0** [comment]
yes，there are also some permission problems:
image

will try to fix tonight.

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**